mypy/test/data.py

```diff
@@ -556,6 +556,9 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    def __init__(self, *, update_data: bool = False) -> None:
```

Comment:
These classes are gathered by pytest, passed to pytest_pycollect_makeitem as the `obj` argument to `pytest_pycollect_makeitem` which returns a `MypyDataSuite` wrapping this `obj`. Later the `collect` method is called on the `MypyDataSuite`, and the (our) implementation iterates over `self.obj.cases()`, wrapping and yielding the results.